### DISCUSSION TITLE
ilogtail是否支持类似 Prometheus 的自动发现配置

### DISCUSSION THREAD

**Discussion by feranwq** [alibaba/ilogtail#1116] [Help] [answered]
在应用的annotations或者 labels 里配置日志路径, ilogtail自动发现后更新采集配置
或者类似 operator 通过crd来描述采集配置

**Comment by linrunqi08** [comment]
@feranwq  “annotations或者 labels 里配置日志路径, ilogtail自动发现后更新采集配置” 目前还不支持。“ operator 通过crd来描述采集配置” 这个商业版已经支持，但是相应的代码还没有开源出来，这个是在计划中的。方便的话，可以留一下应用场景，我们会去做评估，https://github.com/alibaba/ilogtail/issues/693

**Comment by linrunqi08** [comment, marked as answer]
@feranwq  “annotations或者 labels 里配置日志路径, ilogtail自动发现后更新采集配置” 目前还不支持。“ operator 通过crd来描述采集配置” 这个商业版已经支持，但是相应的代码还没有开源出来，这个是在计划中的。方便的话，可以留一下应用场景，我们会去做评估，https://github.com/alibaba/ilogtail/issues/693
